dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -376,4 +428,293 @@ public void zipTempDirectoryToStream(OutputStream out) throws IOException {
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+
+			/* image */
+			_list = ImageLocalManagerUtil.getImages();
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Image.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* portlet */
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+			dc.setSQL("select * from portlet");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Portlet.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* portlet_preferences */
+
+			try{
+				_list = PortletPreferencesLocalManagerUtil.getPreferences();
+			}catch(Exception e){
+				Logger.error(this,"Error in retrieveing all portlet preferences");
+			}
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Portletpreferences.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+
+			//backup content types
+			File file = new File(backupTempFilePath + File.separator + "ContentTypes-" + ContentTypeImportExportUtil.CONTENT_TYPE_FILE_EXTENSION);
+			new ContentTypeImportExportUtil().exportContentTypes(file);
+
+			//backup workflow
+			file = new File(backupTempFilePath + File.separator + "WorkflowSchemeImportExportObject.json");
+			WorkflowImportExportUtil.getInstance().exportWorkflows(file);
+
+			//Backup Rules.
+			file = new File(backupTempFilePath + File.separator + "RuleImportExportObject.json");
+			RulesImportExportUtil.getInstance().export(file);
+
+		} catch (HibernateException e) {
+			Logger.error(this,e.getMessage(),e);
+		} catch (SystemException e) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 711, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)